CMakeLists.txt

```diff
@@ -139,6 +139,16 @@ set(_SHARED_LIB_SUFFIX
 set(_SUNDIALS_ALIAS_TARGETS ""
     CACHE INTERNAL "" FORCE)
 
+# We default to release builds
```

Comment:
Remove the obsolete `if(USE_XSDK_DEFAULTS)` block in `SundialsBuildOptionsPre.cmake` for setting the build type.